src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -603,13 +610,13 @@ private void commitContractCache(Deposit deposit) {
   }
 
   private void commitStorageCache(Deposit deposit) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbJU6pnSgN4uCclo1b1&open=AWbJU6pnSgN4uCclo1b1)

[ISSUE_KEY]: AWbJU6pnSgN4uCclo1b1